webpack.config.common.js

```diff
@@ -68,7 +68,8 @@ module.exports = (env) => ({
                 'css/flow-ui-bootstrap-components': 'bootstrap3',
             },
             bundleFilename: 'bundles.json',
-            pathPrefix: '/',
+            // eslint-disable-next-line no-template-curly-in-string
+            pathPrefix: '/runtime/${BAMBOO_DEPLOY_VERSION}/',
```

Comment:
We may be moving from Bamboo to Jenkins so maybe this should just be named more generic like VERSION.

Reply:
Done :)